### ISSUE TITLE
Video button for a tool bar

### ISSUE THREAD

**Issue by Dashkinru** [a34alefeliza/ile-pikimoni#31]
Tell me how possible it is)) But I think it will be super useful))
I'm asking you to add one more button to a tool bar near the music button - a video button. With that of course to we need a video collection next to the text and audio (for exmample here in the printscreen). It is needed for my grammar video prompts. I can upload them in the storage (they will be pretty small 30 sec or 1 min. Or maybe to insert them with the help of url?!
So the idea is when a teacher opens the "page collection" button, they see not only the collections themselves but also video-promts which have explanations for this grammar. 
Fill free to tell me that you hate me)))
![image](https://user-images.githubusercontent.com/66912801/102823977-f8c5fd80-43ec-11eb-95c9-1c2266bcc102.png)

**Comment by a34alefeliza** [comment]
Yeah, it should be quite simple, as it is very similar to what we already do for Audio. First you create a Video collection (we already have it on some pages!) then using this button on the toolbar you have the possibility to play the video. The video should popup as an overlay (similar to feeds or corrections)? Or somewhere in the page?

**Comment by Dashkinru** [comment]
If it will work as an overlay, it means both the teacher and the student can watch it at the same time, right? But the teacher won't be able to stop it, will them?

**Comment by a34alefeliza** [comment]
correct, the teacher won't be able to stop it. I could not find the way yo make it possible. By overlay, I mean that the page is grayed out and the video will popup ad play on top of the page.

**Comment by Dashkinru** [comment]
And if the video is somewhere on the page, as you have assumed , we won't
have the possibility to control it either, right?

ср, 23 дек. 2020 г., 17:14 Alessandro <notifications@github.com>:

**Comment by a34alefeliza** [comment]
I will double check the possibility to hide the controls to the student
